zapier/introduction/zaps-ii/zapier-data.md

```diff
@@ -0,0 +1,37 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Zapier Data
+
+---
+## Content
+
+Until now, we have only used simple actions in our zaps (monitoring some data or sending a message). Zapier is not limited to that, with another popular application being finding data and using it at a later time.
+
+We have somewhat done this, when we used the filter step to verify that someone has actually gained access to our base.
+
+Now, what if we want to know who has gained access to our base? This is entirely possible, and since we are using Google Sheets we don't require any extra steps.
```

Comment:
```suggestion
Extending on the previous example, what if we want to know who has gained access to our base? This is possible and, since we are using Google Sheets, we don't need to add any extra steps in the zap.
```

^ wondering if "task" works better than "step" 

also, it might be adding a paragraph above this where we briefly describe the previous example. if someone will read these 2 workouts (zaps-i & zaps-ii) some time apart then this will be quite confusing

Reply:
agreed, I'll add it as a footnote